pkg/maintenance/interfaces.go

```diff
@@ -0,0 +1,9 @@
+package maintenance
+
+import (
+	"github.com/cloudogu/cesapp-lib/registry"
+)
+
+type globalConfig interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `globalConfig` is unused